Package/Sublime Text Build System/Sublime Build System.sublime-syntax

```diff
@@ -0,0 +1,400 @@
+        - match: \"
+          scope: punctuation.definition.string.end.json punctuation.definition.placeholder-string.end.sublime-build
+          pop: true
+        - match: $\n?
+          scope: invalid.illegal.unclosed-string.json
+          pop: true
+        - include: string-placeholder
+
+  expect-working-dir-value:
+    - match: \"
+      scope: punctuation.definition.string.begin.json punctuation.definition.placeholder-string.begin.sublime-build
+      set:
+        - meta_scope: meta.value.json meta.placeholder-string.sublime-build string.quoted.double.json
+        - match: \"
+          scope: punctuation.definition.string.end.json punctuation.definition.placeholder-string.end.sublime-build
+          pop: true
+        - match: $\n?
+          scope: invalid.illegal.unclosed-string.json
+          pop: true
+        - include: string-placeholder
+
+  string-placeholder:
+    - match: '\\[$}]'
+      scope: constant.character.escape.placeholder.sublime-build
+    - match: '(\$)(\{)(?:({{placeholder}})\b|(\w*))'
+      captures:
+        1: keyword.other.block.start.placeholder.sublime-build
+        2: keyword.other.block.start.placeholder.sublime-build
+        3: meta.placeholder-name.sublime-build constant.other.placeholder.sublime-build
+        4: meta.placeholder-name.sublime-build
+      push:
+        - meta_content_scope: meta.placeholder-name.sublime-build
+        - match: \}
+          scope: keyword.other.block.end.placeholder.sublime-build
+          pop: true
+        - match: ':'
+          scope: punctuation.separator.placeholder.alternative.placeholder.sublime-build
+          set:
+            - match: \}
+              scope: keyword.other.block.end.placeholder.sublime-build
+              pop: true
+            - include: string-placeholder
+        - match: '/'
+          scope: punctuation.separator.placeholder.regex-replace.begin.placeholder.sublime-build
+          set:
+            - match: \}
+              scope: invalid.illegal.expected-slash.regex-replace.placeholder.sublime-build
+              pop: true
+            - match: '/'
+              scope: punctuation.separator.placeholder.regex-replace.mid.placeholder.sublime-build
+              set:
+              - match: \}
+                scope: invalid.illegal.expected-slash.regex-replace.placeholder.sublime-build
+                pop: true
+              - match: '/'
+                scope: punctuation.separator.placeholder.regex-replace.end.placeholder.sublime-build
+                set:
+                  - match: \}
+                    scope: keyword.other.block.end.placeholder.sublime-build
+                    pop: true
+                  - match: '[gimsx-]*'
+                    scope: constant.other.regex-modifier.placeholder.sublime-build
+                  - match: \S
+                    scope: invalid.illegal.unexpected-token.placeholder.sublime-build
+              - include: scope:source.regexp-replacement
```

Comment:
I believe we only have this in the legacy format. It's listed in #67 (escaped boost regex).